app/index.html

```diff
@@ -448,6 +451,24 @@ <h3>Add Workflow from Catalog</h3>
             </div>
         </div>
 
+        <div id="import-catalog-object-confirmation-modal" class="modal fade" tabindex="-1" role="dialog" aria-hidden="true">
+            <div class="modal-dialog">
+                <div class="modal-content">
+                    <div class="modal-header">
+                        <button type="button" class="close" data-dismiss="modal" aria-hidden="true">×</button>
+                        <h3>Import from Catalog</h3>
+                    </div>
+                    <div class="modal-body" style="height:100%">
+                        You are about to import an object from the Catalog. If you continue it will replace and remove its current value in the text area.
```

Comment:
You are about to import an object from Catalog (without the, since Catalog is a known entity and not a simple noun)

Reply:
If you continue, it will replace ... (add a comma)

Reply:
@amouhoub, actually, I would use "the" everywhere. 
If we want to use "Catalog" as a known entity, we would use then "ProActive Catalog". Like "publish your application on Google Play".
Here we can use "the" because we are in the context of the same product. The product has two components, "the" Studio and "the" Catalog.

Reply:
I agree with you Fabien, we say "the dashboard", "the studio" everywhere